### ISSUE TITLE
add withdrawl argument to nhanesSearchTableNames

### ISSUE THREAD

**Issue by Ifitrains** [cjendres1/nhanes#15]
`nhanesA::nhanesSearchTableNames("PAHS", details = TRUE)` returns, 

```
      Years                                                  Data.File.Name   Doc.File
1 2011-2012        Polyaromatic Hydrocarbons (PAHs)- Urine - Special Sample PAHS_G Doc
2 2013-2014 Polycyclic Aromatic Hydrocarbons (PAH) - Urine - Special Sample PAHS_H Doc
3 2015-2016 Polycyclic Aromatic Hydrocarbons (PAH) - Urine - Special Sample PAHS_I Doc
                     Data.File Date.Published
1   PAHS_G Data [XPT - 426 KB]  February 2014
2 PAHS_H Data [XPT - 354.6 KB]      **Withdrawn**
3 PAHS_I Data [XPT - 310.7 KB]      July 2020
```

Withdrawn datasets are not really useful and cannot be downloaded from the `nhanes()` function, so it would be very nice to have an option to automatically remove withdrawn datasets from the search.

**Comment by cjendres1** [comment]
Thanks for submitting this. I'll take a look but may not get to it until after Thanksgiving.

**Comment by cjendres1** [comment]
Just pushed a fix to git. 
nhanesSearchTableNames now has option `includewithdrawn`.

Please install the update using: `install_github("cjendres1/nhanes")`
It will install as nhanesA 0.7.2

By default, includewithdrawn=FALSE. Thus Withdrawn tables should not show up when you run your previous command.
`nhanesSearchTableNames("PAHS", details = TRUE)`

Please test the function with these commands:
```
nhanesSearchTableNames("PAHS",  includewithdrawn = TRUE, details = TRUE) 
nhanesSearchTableNames("PAHS",  includewithdrawn = FALSE, details = TRUE)
```

**Comment by cjendres1** [comment]
Fix included in nhanesA 0.7.2
